apps/console/src/features/applications/components/wizard/passive-sts-protocol-settings-wizard-form.tsx

```diff
@@ -141,6 +142,45 @@ export const PassiveStsProtocolSettingsWizardForm: FunctionComponent<PassiveStsS
                                 </Hint>
                             </Grid.Column>
                         </Grid.Row>
+                        <Grid.Row>
+                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                                <Field
+                                    name="replyToLogout"
+                                    label={
+                                        t("console:develop.features.applications.forms.inboundSTS.fields." +
+                                          "replyToLogout.label")
+                                    }
+                                    required={ true }
+                                    requiredErrorMessage={
+                                        t("console:develop.features.applications.forms.inboundSTS.fields." +
+                                          "replyToLogout.validations.empty")
+                                    }
+                                    placeholder={
+                                        t("console:develop.features.applications.forms.inboundSTS.fields." +
+                                          "replyToLogout.placeholder")
+                                    }
+                                    validation={ (value: string, validation: Validation) => {
+                                        if (!FormValidation.url(value)) {
+                                            validation.isValid = false;
+                                            validation.errorMessages.push(
+                                                t("console:develop.features.applications.forms" +
+                                                  ".inboundSTS.fields.replyToLogout.validations.invalid")
+                                            );
+                                        }
+                                    } }
+                                    type="text"
+                                    value={ initialValues
+                                        ? initialValues?.replyToLogout
+                                        : templateValues?.replyToLogout
```

Comment:
Let's try to simplify this like,
```suggestion
                                    value={ initialValues?.replyToLogout ?? templateValues?.replyToLogout
```

Reply:
Addressed with 37333d9